cli/azd/cmd/cobra_builder.go

```diff
@@ -148,64 +143,11 @@ func (cb *CobraBuilder) configureActionResolver(cmd *cobra.Command, descriptor *
 		// Set the container that should be used for resolving middleware components
 		runOptions.WithContainer(cmdContainer)
 		// Run the middleware chain with action
-		actionResult, err := middlewareRunner.RunAction(ctx, runOptions, action)
+		_, err = middlewareRunner.RunAction(ctx, runOptions, action)
 
 		// At this point, we know that there might be an error, so we can silence cobra from showing it after us.
 		cmd.SilenceErrors = true
 
-		// TODO: Consider refactoring to move the UX writing to a middleware
-		invokeErr := cmdContainer.Invoke(func(console input.Console) {
```

Comment:
I was never a fan of this `Invoke(...)` statement just to get the `console` object that we should never have needed to use here.  This piece has been moved to the new UX middleware.